src/store/actions/actions.js

```diff
@@ -59,28 +59,29 @@ export const deselectItems = () => {
   };
 };
 
-export const resetSelectedProject = () => {
+export const processDataIfReady = () => {
   return {
-    type: actionTypes.RESET_SELECTED_PROJECT
+    type: actionTypes.PROCESS_DATA_IF_READY
   };
 };
 
-export const deactivatePopover = () => {
+export const resetSelectedProject = () => {
   return {
-    type: actionTypes.DEACTIVATE_POPOVER
+    type: actionTypes.RESET_SELECTED_PROJECT
   };
 };
 
-export const getFiltersFromURL = () => {
+export const deactivatePopover = () => {
   return {
-    type: actionTypes.GET_FILTERS_FROM_URL
+    type: actionTypes.DEACTIVATE_POPOVER
   };
 };
 
 export const fetchClusterData = () => {
   return dispatch => {
-    axios.get("/api/clustering").then(result => {
+    axios.get("./cluster.json").then(result => {
```

Comment:
can you turn this into the api/clustering string again? otherwise everything seems fine with me :)